templates/qa_zre/templates.yaml

```diff
@@ -0,0 +1,50 @@
+dataset: qa_zre
+templates:
+  2d6b6ec6-4cba-4a07-a0d1-f6b7cb103281: !Template
+    id: 2d6b6ec6-4cba-4a07-a0d1-f6b7cb103281
+    jinja: 'Extract the appropriate relation from the following question
+
+
+      {{question}} |||
+
+      {{relation}}'
+    name: relation
+    reference: https://www.aclweb.org/anthology/K17-1034.pdf
+  5a970b88-53a0-4148-b45e-7ac410df263f: !Template
+    id: 5a970b88-53a0-4148-b45e-7ac410df263f
+    jinja: "What is a possible question that can be generated from the following context\
+      \ and answer(s)?\n\n{{context}} \n\n{{answers|join(\", \")}} |||\n{{question|replace(\"\
```

Comment:
It's a little confusing to say "and answer(s)" for the examples where there are no answers. One option would be to only say "and answer(s)" when answers is non-empty, but that would create duplicates with the possible_qn template. I think it's probably best to modify the "possible_qn_with_answer" template so that it only includes "and answer(s)" when answers is non-empty and remove the "possible_qn" template.